howto/stream/exchange-oob-data.md

```diff
@@ -9,9 +9,9 @@ Anbox Cloud provides two versions of this OOB data exchange:
 * [Version 2](#version-2) provides a full-duplex bidirectional data transmission mode in which data can flow in both directions at the same time.
 
   Use this version if you start your implementation now. If you already have an existing implementation, you should plan to update it to use version 2.
-* [Version 1](#version-1) enables Android application developers to trigger an action from an Android application running in an instance and forward it to a WebRTC client through the Anbox WebRTC platform. When Anbox receives the action, as one peer of the WebRTC platform, the action is propagated from Anbox to the remote peer (the WebRTC client) through a WebRTC data channel. The client can then react to the action received from the remote peer and respond accordingly on the UI.
+* [Version 1](#version-1) enables Android application developers to trigger an action from an Android application running in an instance and forward it to a WebRTC client through the Anbox runtime. When Anbox receives the action, as one peer of the WebRTC platform, the action is propagated from Anbox to the remote peer (the WebRTC client) through a WebRTC data channel. The client can then react to the action received from the remote peer and respond accordingly on the UI.
```

Comment:
As support for version 1 has been removed (see below) shoud we not get rid of this here?

Reply:
You mean drop the Version 1 complete in the section?
I'm fine with that. What do you think @keirthana ?

Reply:
It's been a while since we removed support, I am fine with removing it.